### PR TITLE
Get tests passing for PHP7

### PR DESCRIPTION
**Pull request by zackgalbreath**
These changes get the whole test suite passing for MySQL & Postgres using PHP7

### CODE REVIEW

File: tests/singletest.php [hunk 1]
```diff
@@ -17,4 +17,5 @@
 $manager->removeLogAndBackupFiles($logfilename);
 //$manager->setTestDirectory(getcwd());
 $manager->setDatabase($db);
-$manager->runFileTest(new TextReporter(), $argv[1]);
```
Comment (mgrauer) [inline tests/singletest.php]: what was the warning?  It didn't like that it was an anonymous (no-handle) object?

**Comment by mgrauer** [comment]
Default sort ordering for identical objects changing is a bit disturbing.  I could see that breaking something and being very hard to track down.

LGTM.

File: tests/singletest.php (hunk 1 above)
Comment (zackgalbreath) [inline tests/singletest.php]: "Only variables should be passed by reference"

**Comment by zackgalbreath** [comment]
> Default sort ordering for identical objects changing is a bit disturbing. I could see that breaking something and being very hard to track down.

Agreed.  Hopefully we aren't relying on such behavior anywhere else.

To their credit, PHP did mention this in their [release notes](http://php.net/manual/en/migration70.incompatible.php).


> Sort order of equal elements
> 
> The internal sorting algorithm has been improved, what may result in different sort order of elements, which compare as equal, than before.
> 
>Note:
> Don't rely on the order of elements which compare as equal; it might change anytime.

**Pull request merged**